data/dummy_data.py

```diff
@@ -0,0 +1,45 @@
+# import pandas as pd
+# from faker import Faker
+# from collections import defaultdict
+# # # from sqlalchemy import create_engine
+
+# fake = Faker()
+
+# fake_data = defaultdict(list)
+
+# for _ in range(100):
+#     fake_data["first_name"].append( fake.first_name() )
+#     fake_data["last_name"].append( fake.last_name() )
+#     fake_data["occupation"].append( fake.job() )
+#     fake_data["dob"].append( fake.date_of_birth() )
+#     fake_data["country"].append( fake.country() )
+
+# df_fake_data = pd.DataFrame(fake_data)
+
+# df_fake_data.shape
```

Comment:
Remove commented code in the final commits accompanying a PR. If you need to refer to them later, you can refer to specific commits.